app/controllers/api/v1/demographics/districts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::Demographics::DistrictsController < Api::ApiController
+  respond_to :json
+
+  def show
```

Comment:
This is on my radar, but I think how I handle errors will change as I learn more about best practices, so I'm putting this aside until I lock in my implementation